README.md

```diff
@@ -1 +1,18 @@
-# customerio-reactnative
\ No newline at end of file
+# Customer.io ReactNative SDK
+
+This is the official Customer.io SDK for ReactNative.
+
+# More information
+
+See our complete SDK documentation at [https://customer.io/docs/sdk/react-native/](https://customer.io/docs/sdk/react-native/)
+
+# Contributing
+
+Thanks for taking an interest in our project! We welcome your contributions. Check out [our development instructions](docs/dev-notes/DEVELOPMENT.md) to get your environment set up and start contributing.
+
+> **Note:**
+> We value an open, welcoming, diverse, inclusive, and healthy community for this project. We expect all  contributors to follow our [code of conduct](https://github.com/customerio/mobile/blob/HEAD/CODE_OF_CONDUCT.md).  
+
+# License
+
+[MIT](https://github.com/customerio/mobile/blob/HEAD/LICENSE)
```

Comment:
```suggestion
[MIT](LICENSE)
```

I like the idea of referring to 1 centralized LICENSE file, but I think that in open source development, legally, a project isn't open source unless you include a LICENSE file inside of the actual source code. Which, the RN SDK does have one in it's repo. I suggest that we lint to that file in the repo and not an external one.

Reply:
I completely forgot about it 🤦🏻 Thanks for pointing this out. I'll copy license to this repo.